.github/workflows/stale.yml

```diff
@@ -44,22 +47,51 @@ jobs:
 
             Thank you for your contributions.
 
-      # The 30 day stale policy
+      # The 30 day stale policy for PRS
       # Used for:
-      # - Issues that are pending more information (incomplete issues)
-      # - PRs that are not marked as new-integration
-      - name: 30 days stale policy
+      # - PRs
+      # - No PRs marked as no-stale or new-integrations
+      # - No issues (-1)
+      - name: 30 days stale PRs policy
         uses: actions/stale@v3.0.15
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           # PRs have a CLA signed label, we can misuse it to apply this policy
-          only-labels: "cla-signed,needs-more-information"
           days-before-stale: 30
           days-before-close: 7
-          operations-per-run: 250
+          days-before-issue-close: -1
+          operations-per-run: 50
+          remove-stale-when-updated: true
+          stale-pr-label: "stale"
+
+          # Exempt new integrations, these often take more time.
+          # They will automatically be handled by the 90 day version above.
+          exempt-pr-labels: "no-stale,new-integration"
+          stale-pr-message: >
+            There hasn't been any activity on this pull request recently. This
+            pull request has been automatically marked as stale because of that
+            and will be closed if no further activity occurs within 7 days.
+
+            Thank you for your contributions.
+
+      # The 30 day stale policy for issues
+      # Used for:
+      # - Issues that are pending more information (incomplete issues)
+      # - No Issues marked as no-stale or help-wanted
+      # - No PRs (-1)
+      - name: Needs more information stale issues policy
+        uses: actions/stale@v3.0.15
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # PRs have a CLA signed label, we can misuse it to apply this policy
```

Comment:
Yet you aren't using said label?

Reply:
Right left over from the old policy (as a comment), well spotted 👍

Reply:
Adjusted, thanks! 👍